plugin/api.go

```diff
@@ -68,6 +68,10 @@ type API interface {
 	// The status parameter can be: "online", "away", "dnd", or "offline".
 	UpdateUserStatus(userId, status string) (*model.Status, *model.AppError)
 
+	// GetUsersInChannel returns a page of users in a channel. Page counting starts at 0.
+	// The sortBy parameter can be: "username" or "status"
```

Comment:
Can you document the minimum version required here?

```suggestion
	// The sortBy parameter can be: "username" or "status".
	//
	// Minimum server version: 5.6
```